src/atomate2/vasp/schemas/phonons.py

```diff
@@ -0,0 +1,474 @@
+        None,
+        description="field including thermal displacement matrices in cartesian coordinate system",
+    )
+    temperatures_thermal_displacements: List[int] = Field(
+        None,
+        description="temperatures at which the thermal displacement matrices"
+        "have been computed",
+    )
+
+
+class PhononUUIDs(BaseModel):
+    """
+    Collection to save all uuids connected to the phonon run
+    """
+
+    optimization_run_uuid: str = Field(None, description="optimization run uuid")
+    displacements_uuids: List[str] = Field(
+        None, description="The uuids of the displacement jobs."
+    )
+    static_run_uuid: str = Field(None, description="static run uuid")
+    born_run_uuid: str = Field(None, description="born run uuid")
+
+
+class PhononJobDirs(BaseModel):
+    """
+    Collection to save all job directories relevant for the phonon run
+    """
+
+    displacements_job_dirs: List[str] = Field(
+        None, description="The directories where the displacement jobs were run."
+    )
+    static_run_job_dir: str = Field(
+        None, description="Directory where static run was performed."
+    )
+    born_run_job_dir: str = Field(
+        None, description="Directory where born run was performed."
+    )
+    optimization_run_job_dir: str = Field(
+        None, description="Directory where optimization run was performed."
+    )
+
+
+class PhononBSDOSDoc(BaseModel):
+    """
+    Collection of all data produced by the phonon worklow
+    """
+
+    structure: Structure = Field(
+        None,
+        description="Structure of Materials Project.",
+    )
+
+    ph_bs: PhononBandStructureSymmLine = Field(
+        None,
+        description="Phonon band structure object.",
+    )
+
+    ph_dos: PhononDos = Field(
+        None,
+        description="Phonon density of states object.",
+    )
+
+    free_energy_list: List[float] = Field(
+        None,
+        description="vibrational part of the free energies in kJ/mol per "
+        "formula unit for temperatures in temperature_list",
+    )
+
+    temperatures: List[int] = Field(
+        None,
+        description="temperatures at which the vibrational"
+        " part of the free energies"
+        " and other properties have been computed",
+    )
+
+    total_energy: float = Field("total DFT energy per formula unit in eV")
+
+    has_imaginary_modes: bool = Field(
+        None, description="if true, structure has imaginary modes"
+    )
+
+    # needed, e.g. to compute Grueneisen parameter etc
+    force_constants: List[List[Matrix3D]] = Field(
+        None, description="Force constants between every pair of atoms in the structure"
+    )
+
+    born: List[Matrix3D] = Field(
+        None,
+        description="born charges as computed from phonopy. Only for symmetrically different atoms",
+    )
+
+    epsilon_static: Matrix3D = Field(
+        None, description="The high-frequency dielectric constant"
+    )
+    dft_phononwebsite: dict = Field(
```

Comment:
Do you have an idea of the size of this data? Is it very big?

Reply:
Will check this! Thanks for the hint!

Reply:
I decided to simply write a file to the last folder including this information so that it does not enlarge the database. Currently, this is only a dict and it has to be of the same size as the bandstructure objects.